src/Bridges/Nette/TracyExtension.php

```diff
@@ -19,6 +19,8 @@ class TracyExtension extends Nette\DI\CompilerExtension
 {
 	public $defaults = array(
 		'email' => NULL,
+		'logDirectory' => NULL,
+		'logSeverity' => 0,
```

Comment:
To rewrite what user sets in boostrap.php is bad. https://github.com/JanTvrdik/nette-tracy/blob/f47bdf5dccfdc26dbb61cf7c4cf1f128ef2ffd7e/src/Bridges/Nette/TracyExtension.php#L71